VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-context-menu/judge-context-menu.component.ts

```diff
@@ -120,4 +120,12 @@ export class JudgeContextMenuComponent {
             ? this.translateService.instant('judge-context-menu.unpin')
             : this.translateService.instant('judge-context-menu.pin');
     }
+
+    callParticipantIntoHearing(): boolean {
+        return (this.participant.isWitness || this.participant.isQuickLinkUser) && !this.participant.isInHearing();
+    }
+
+    dismissParticipantFromHearing(): boolean {
+        return (this.participant.isWitness || this.participant.isQuickLinkUser) && this.participant.isInHearing();
+    }
```

Comment:
I think these should be clearer that they are checks and not actions. canCallParticipantIntoHearing and canDismissParticipantFromHearing. Also maybe a private function for the (this.participant.isWitness || this.participant.isQuickLinkUser) part so it's easier to maintain.

Reply:
Done